bin/detail/toolchain_table.py

```diff
@@ -315,6 +317,13 @@ def verify(self):
           arch='amd64',
           vs_version='15'
       ),
+      Toolchain(
+          'vs-15-2017-win64-llvm',
+          'Visual Studio 15 2017 Win64',
+          toolset='LLVM-vs2014',
```

Comment:
I haven't used this yet, don't know the details. So can we do `LLVM-vs2017` too? In this case toolchain name should be `vs-15-2017-win64-llvm-2014` and `vs-15-2017-win64-llvm-2017`.

Reply:
Clang 6.0 (latest full release) only provides a toolset named LLVM-vs2014. I can name the toolchain accordingly but there is no LLVM-vs2017